prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var firstElement = languages[0];
+console.log('The first element of the array is:', firstElement);
+var thirdElement = languages[2];
+console.log('The third element of the array is:', thirdElement);
+var length = languages.length;
+console.log('The length of the array is:', length);
+var lastElement = languages[5];
```

Comment:
You can get the last element of any array by subtracting `1` from its length without having to know for sure how long it it.

```js
var lastIndex = languages.length - 1;
```

Then you can use that variable to get the last element

```js
var lastLanguage = languages[lastIndex];
```

Reply:
![prep-javascript-arrays screenshot](https://user-images.githubusercontent.com/61856201/156256839-bc9685aa-27e5-48e3-a04b-cfd4bb4ff21d.png)